src/test/java/org/recap/service/executor/datadump/DeletedDataDumpExecutorServiceUT.java

```diff
@@ -210,11 +107,12 @@ public void testpopulateSearchRequest() {
         searchRecordsRequest.setIndex(1);
         searchRecordsRequest.setDeleted(false);
         searchRecordsRequest.setErrorMessage("test");
-        try{
-        mockedDeletedDataDumpExecutorService.populateSearchRequest(searchRecordsRequest, dataDumpRequest);
-        } catch (Exception e) {
-        e.printStackTrace();
-        }
-        assertTrue(true);
+        return searchRecordsRequest;
+    }
+
+    private String getDateTimeString() {
+        Date date = new Date();
```

Comment:
I see the same method in the previous class - duplicate code try to minimize that